src/v2/Apps/Order/Routes/Review/index.tsx

```diff
@@ -376,9 +377,17 @@ export class ReviewRoute extends Component<ReviewProps> {
             <Flex flexDirection="column">
               <Flex flexDirection="column">
                 <ArtworkSummaryItem order={order} />
-                <TransactionDetailsSummaryItem order={order} />
+                <TransactionDetailsSummaryItem
+                  order={order}
+                  placeholderOverride="To be confirmed*"
+                />
               </Flex>
               <BuyerGuarantee />
+              {!order.myLastOffer?.hasDefiniteTotal && (
```

Comment:
@lilyfromseattle @xtina-starr Sorry to lose track of this PR. Would this also render the message for Buy Now orders? Maybe we need something like this?
```typescript
order.myLastOffer && !order.myLastOffer.hasDefiniteTotal
```